src/reducers/businessCaseReducer.ts

```diff
@@ -61,6 +61,7 @@ function businessCaseReducer(
     case postBusinessCase.FULFILL:
       return {
         ...state,
+        isLoading: false,
```

Comment:
does this mean we should set isLoading to true in REQUEST? Also, do we need to have a FAILURE case for this now that it's necessary to move forward?

Reply:
Hmmm, I'm not sure about the `isLoading`. I'm not sure what is semantically correct, but I don't think any behavior will change either way right now.

Reply:
then why do we change it to false?

Reply:
I changed it to `false` because there was a wrapper around the form that only displayed if `isLoading === false`. I changed that now though.